SEED-platform/seed
# feat: add derived columns to exported results

#### Any background context you want to provide?
SEED is adding the feature for users to define columns calculated from other columns (derived columns)

#### What's this PR do?
Adds derived columns and their values to exported sheets (CSV, excel) for derived columns part of the column profile they're using.

#### How should this be manually tested?
1. create derived column, have property that includes required columns
2. create a column list profile which includes the derived column
3. select properties in list view, then choose export
4. you should see the derived column and its data in the resulting file

#### What are the relevant tickets?
#2717 

#### Screenshots (if appropriate)
![Screen Shot 2021-06-15 at 6 58 04 PM](https://user-images.githubusercontent.com/18518728/122142157-15774700-ce0c-11eb-9b9d-c9df147a1405.png)
